src/ucx_van.h

```diff
@@ -48,6 +48,10 @@ std::mutex g_log_mutex;
     ucp_request_free(_req); \
   } while(0)
 
+const int UCX_OPTION_META = -1;
+const int UCX_OPTION_DATA = -2;
```

Comment:
Is it really needed?
`UCX_OPTION_DATA` seems to be unused and `UCX_OPTION_META` seems to be redundant

Reply:
UCX_OPTION_DATA is added for completeness (used in line 494). Therefore:
option = -1 -> meta only
option = -2 -> data
option > 0 -> meta with data

I added this because we want to preserve `meta.val_len`. We may send a pull request with `val_len != 0`.